lib/core/src/lib/avatar/avatar.component.html

```diff
@@ -1,5 +1,5 @@
 <div class="adf-avatar">
-    <img *ngIf="src; else initialsTemplate" class="adf-avatar__image" [src]="src" [alt]="initials" [title]="tooltip" />
+    <img (error)="onImageError()" *ngIf="src; else initialsTemplate" class="adf-avatar__image" [src]="src" [alt]="initials" [title]="tooltip" />
```

Comment:
onImageError is single line function so no need for extra function. This can be done in this way: 
(error)="src = ''"